lib/main_micro_alp.pm

```diff
@@ -46,7 +46,8 @@ sub load_config_tests {
 
 sub load_boot_from_disk_tests {
     loadtest 'installation/bootloader_start' if is_s390x();
-    if (check_var('FIRST_BOOT_CONFIG', 'wizard')) {
+    if (check_var('HDD_2', '') || check_var('NUMDISK', '1')) {
```

Comment:
This is limiting as you can have more disks set for other tests. Better would be either rely on `FIRST_BOOT_CONFIG` or to check whether hdd2 isn't set to combustion or ignition images.